pkg/controller/jaeger/jaeger_controller_test.go

```diff
@@ -145,6 +146,9 @@ func TestSkipOnNonOwnedCR(t *testing.T) {
 
 	// the only way to reliably test this is to verify that the operator didn't attempt to set the ownership field
 	assert.Equal(t, "another-identity", persisted.Labels[v1.LabelOperatedBy])
+
+	// condition shouldn't be touched when the object belongs to another operator
```

Comment:
not sure what is meant by this comment - what is "condition" referring to? why does the object belong to another operator?

Reply:
In previous iterations, I added another new type, `JaegerCondition`, as an array in the `JaegerStatus`, similar to what other Kubernetes objects have. It serves as a "log" based on the last observation of the object, but I think we don't need it right now.

> why does the object belong to another operator?
This is related to the case described in #567. This test is asserting that we don't change the phase of a CR owned by another operator instance.